README.md

```diff
@@ -21,6 +21,10 @@ Or to download a per-host disk, run:
 
     hammer bootdisk host --host client.example.com
 
+Or to download a subnet disk, run:
+
+    hammer bootdisk subnet --host client.example.com
```

Comment:
> Isn't this `--subnet` or `--subnet-id`?

Indeed, both should work. Copy&paste error. Changed.

-- 
Later,
 Lukas #lzap Zapletal